backend/service.cpp

```diff
@@ -163,83 +162,100 @@ ogon_incoming_bytes_result ogon_service_incoming_bytes(ogon_backend_service *ser
-	switch (service->messageType)
-	{
-	case OGON_CLIENT_CAPABILITIES:
-		IFCALLRET(client->Capabilities, success, cb_data, &msg->capabilities);
-		break;
-	case OGON_CLIENT_SYNCHRONIZE_KEYBOARD_EVENT:
-		IFCALLRET(client->SynchronizeKeyboardEvent, success, cb_data, msg->synchronizeKeyboard.flags,
-				  msg->synchronizeKeyboard.clientId);
-		break;
-	case OGON_CLIENT_SCANCODE_KEYBOARD_EVENT:
-		IFCALLRET(client->ScancodeKeyboardEvent, success, cb_data, msg->scancodeKeyboard.flags,
-				  msg->scancodeKeyboard.code, msg->scancodeKeyboard.keyboardType, msg->scancodeKeyboard.clientId);
-		break;
-	case OGON_CLIENT_UNICODE_KEYBOARD_EVENT:
-		IFCALLRET(client->UnicodeKeyboardEvent, success, cb_data, msg->unicodeKeyboard.flags, msg->unicodeKeyboard.code,
-				  msg->unicodeKeyboard.clientId);
-		break;
-	case OGON_CLIENT_MOUSE_EVENT:
-		IFCALLRET(client->MouseEvent, success, cb_data, msg->mouse.flags, msg->mouse.x, msg->mouse.y,
-				  msg->mouse.clientId);
-		break;
-	case OGON_CLIENT_EXTENDED_MOUSE_EVENT:
-		IFCALLRET(client->ExtendedMouseEvent, success, cb_data, msg->extendedMouse.flags, msg->extendedMouse.x,
-				  msg->extendedMouse.y, msg->extendedMouse.clientId);
-		break;
-	case OGON_CLIENT_FRAMEBUFFER_SYNC_REQUEST:
-		IFCALLRET(client->FramebufferSyncRequest, success, cb_data, msg->framebufferSyncRequest.bufferId);
-		break;
-	case OGON_CLIENT_SBP_REPLY:
-		IFCALLRET(client->Sbp, success, cb_data, &msg->sbpReply);
-		break;
-	case OGON_CLIENT_IMMEDIATE_SYNC_REQUEST:
-		IFCALLRET(client->ImmediateSyncRequest, success, cb_data, msg->immediateSyncRequest.bufferId);
-		break;
-	case OGON_CLIENT_SEAT_NEW:
-		IFCALLRET(client->SeatNew, success, cb_data, &msg->seatNew);
-		break;
-	case OGON_CLIENT_SEAT_REMOVED:
-		IFCALLRET(client->SeatRemoved, success, cb_data, msg->seatRemoved.clientId);
-		break;
-	case OGON_CLIENT_MESSAGE:
-		IFCALLRET(client->Message, success, cb_data, &msg->message);
-		break;
-	case OGON_CLIENT_VERSION:
-		msgVersion.versionMajor = OGON_PROTOCOL_VERSION_MAJOR;
-		msgVersion.versionMinor = OGON_PROTOCOL_VERSION_MINOR;
-		msgVersion.cookie = getenv("OGON_BACKEND_COOKIE");
-		if (!ogon_service_write_message(service, OGON_SERVER_VERSION_REPLY, (ogon_message *) &msgVersion)) {
-			WLog_ERR(TAG, "failed to write version reply message");
-			success = FALSE;
-		}
-		else if (msg->version.versionMajor != OGON_PROTOCOL_VERSION_MAJOR) {
-			WLog_ERR(TAG, "received protocol version info with %"PRIu32".%"PRIu32" but own protocol version is %"PRIu32".%"PRIu32"",
-				msg->version.versionMajor, msg->version.versionMinor,
-				OGON_PROTOCOL_VERSION_MAJOR, OGON_PROTOCOL_VERSION_MINOR);
-			success = FALSE;
-		}
-		break;
+	/* WLog_DBG(TAG, "message type %"PRIu16" (%s)", service->messageType,
+	 * ogon_message_name(service->messageType)); */
+	switch (service->messageType) {
+		case OGON_CLIENT_CAPABILITIES:
+			IFCALLRET(
+					client->Capabilities, success, cb_data, &msg->capabilities);
```

Comment:
hum pretty ugly code style

Reply:
unfortunately the old code did violate lots of format string requirements (e.g. `"sometext %" PRIx32` requires the space between the format string and the `PRIx32` macro), so the reformatting was required.